descriptors/ModuleDescriptor-template.json

```diff
@@ -616,6 +616,17 @@
       "version": "1.0",
       "interfaceType": "system",
       "handlers": [
+        {
+          "methods": [
+            "POST"
+          ],
+          "pathPattern": "/circulation/rules-reload",
```

Comment:
That is how I originally had it set up, but @alexanderkurash asked me to make the timer a completely separate endpoint.  He had concerns about regressions and also pointed out that all the timers have their own endpoints, so this would be more consistent.

Reply:
Oh that's right, it's a timer. It's ok then.